src/Widgets/PagesList.vala

```diff
@@ -76,13 +76,16 @@ public class ENotes.PagesList : Gtk.Box {
         });
 
         listbox.set_sort_func ((row1, row2) => {
-            int64 a = ((PageItem) row1).page.id;
-            int64 b = ((PageItem) row2).page.id;
+            int64 a = ((PageItem) row1).page.modification_date;
+            int64 b = ((PageItem) row2).page.modification_date;
```

Comment:
For now, leave the sorting as by page.id. This will make sure that the changes won't affect other users, and we can change said sorting by later implementing #118 on a different PR :)